### PR TITLE
feat: added dice Component (Mit Timo und Desi)

### LINKED ISSUE #192: implement roll dice mechanism
# Description
*Short description of the feature*

# Tasks
## Tasks backend

- Implement an endpoint that returns a random number between 1 to 6. 
- Return given book section (Described in: #185) based on result of rolled dice 

## Tasks frontend

- call endpoint, show result of rolled dice

### PR DESCRIPTION
**Pull request by kaibria**
### What?
A component which displays an animated dice
### Why?
Because some section of the book require a dice roll
### How?
With the Libary react-dice-complete
### Testing?
Manual Testing

closes #192

### CODE REVIEW

**Review by tamer545 — APPROVED** [review]
Looks good to me!

**Review by nnikolajj — COMMENTED** [review]
Könnt ihr noch das Issue angeben, das ihr closen werded

**Review by tamer545 — COMMENTED** [review]
The automatic issue close is missing

File: fabbwled-frontend/src/DiceComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {useRef} from 'react';
+import ReactDice, {ReactDiceRef} from "react-dice-complete";
+
+interface DiceComponentProps {
```
Comment (corsin-lehre) [inline fabbwled-frontend/src/DiceComponent.tsx:4]:
```suggestion
type DiceComponentProps {
```
just to streamline how we declare props

**Review by Oliver-was-taken — APPROVED** [review]
looks good, watch corsins requests
Maybe you should also push package-lock

**Review by Oliver-was-taken — APPROVED** [review]
thanks for rerequesting @tamer545 - it still looks good

**Review by corsin-lehre — APPROVED** [review]
4904cda6-feb2-4034-b67d-22050bb1f46d_text

**Review by Noeell — APPROVED** [review]
good work

**Review by JJoeeyy — APPROVED** [review]

**Pull request merged**